tools/confix/go.mod

```diff
@@ -35,7 +35,7 @@ require (
 	github.com/cosmos/cosmos-proto v1.0.0-beta.1 // indirect
 	github.com/cosmos/go-bip39 v1.0.0 // indirect
 	github.com/cosmos/gogogateway v1.2.0 // indirect
-	github.com/cosmos/gogoproto v1.4.3 // indirect
+	github.com/cosmos/gogoproto v1.4.4 // indirect
```

Comment:
https://github.com/cosmos/cosmos-sdk/actions/runs/4047744533/jobs/6962151076
I think our script go mod update all is fundamentally broken, as it updates all the dependencies, regardless if they are actually needed or not.
Here, for instance, we use 0.47-rc1 so it should not have updated this to v1.4.4 (as there is no reason to)

Reply:
Do you have a recommendation for this PR? Bumping all to v1.4.4 seems to break some tests (like confix).

I reverted all of the bumps, then only bumped to v1.4.4 the ones that don't depend on cosmos-sdk@v.047-rc1 (e.g. root, store, api), then ./scripts/go-mod-tidy-all. And committed the result.

In any case, the cleanest way might be to tag rc2 after this PR, and bump every module to rc2+v1.4.4

Reply:
Yes, makes sense. Before rc2, I think we need to, though: https://github.com/cosmos/cosmos-sdk/pull/14818

Reply:
Sounds good, I'll let you decide when's the best time to tag rc2